content/hi/continuous_integration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर इंटीग्रेशन
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+निरंतर इंटीग्रेशन , जिसे  ज्यादातर  सी.आई. बोला जाता है, यह कोड परिवर्तन को  जल्दी से जल्दी  एकीकृत करने का एक अभ्यास है। सी.आई. [सी.डी](/Continuous_Delivery/) पढ़ने के लिए एक ज़रूरी अवधारणा है। ज़्यादातर सी.आई. की प्रक्रिया तब शरू होती है जब कोड परिवर्तन  को स्रोत नियंत्रण सिस्टम (गित, सबवर्सन, और  मरक्यूरिअल) में कमिट किया जाता है और फिर वह ख़तम होता है एक परीक्षण किया हुए उत्पाद की तरह जो की तैयार है सी.डी सिस्टम में जाने के लिए।
+
+## समस्या
+
+ज़्यादातर सॉफ्टवेयर सिस्टम  बहुत  बड़े और जटिल होते है जिसको कई डेवेलपर्स रोज़ मेन्टेन और अपडेट करते है। वह लोग समानांतर अपने सिस्टम पर काम करते है जिस कारण कई कान्फ्लिक्टिंग परिवर्तन आ सकते है और अनजाने में एक दूसरे के कोड में  दिक्कत आ जाती  है। इसके साथ ही विभिन्न डेवेलपर्स रोज़ एक ही प्रोजेक्ट में काम करते है, तो जो रोजमर्रा के कार्य है जैसे कोड का परिक्षण और क्वालिटी की जाँच प्रत्येक डेवलपर द्वारा दोहराया जाना होगा जो की एक समय की बर्बादी होगी।  
```

Comment:
few suggestions here:
- conflicting can be localized to `विवादग्रस्त`
- क्वालिटी can be localized to `गुणवत्ता`

Reply:
Done